feature/google/src/main/java/com/revenuecat/purchases/google/BillingWrapper.kt

```diff
@@ -636,6 +637,38 @@ class BillingWrapper(
         return stringWriter.toString()
     }
 
+    private fun getMappedPurchase(
```

Comment:
This is actually used to obtain the `StoreTransaction` for a `Purchase`. Maybe rename to `getStoreTransaction`